src/index.css

```
@@ -1,22 +1,130 @@
 @import "tailwindcss";
+@import "tw-animate-css";
+@import "shadcn/tailwind.css";
+@import "@fontsource-variable/geist";
 
-@theme {
-  --color-light: #ffffff;
-  --color-dark: #171717;
+@custom-variant dark (&:is(.dark *));
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@custom-variant`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)